bin/ext_service/create_party_leaders_in_db.py

```diff
@@ -0,0 +1,35 @@
+import logging
+import emission.core.get_database as edb
+
+if __name__ == '__main__':
+    logging.basicConfig(level=logging.DEBUG)
+
+    ju_email = "ju.uemura@gmail.com"
+    ju_uuid = edb.get_uuid_db().find_one({'user_email': ju_email})['uuid']
+    logging.debug("Found Juliana's uuid %s" % ju_uuid)
+    edb.get_habitica_db().insert({'habitica_id': "e26ef1c3-d45e-464a-998c-729d107ecc42", 
```

Comment:
if this is what we are doing, don't we need `register` to be idempotent? I am not sure that habitica's register is idempotent. In particular, what happens when I go to the goals screen for the first time? I have never registered the user. Will it prompt me to register? Will that pass?